packages/veritone-react-common/src/components/StructuredDataEngineOutput/story.js

```diff
@@ -0,0 +1,176 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+import styles from './story.styles.scss';
+import StructuredDataEngineOutput from './';
+
+storiesOf('StructuredDataEngineOutput', module).add('Base', () => {
+  let engines = [
+    {
+      id: 'engineOfInterest',
+      name: 'SD Engine Of Interest'
+    }
+  ];
+  let mockEngineOutput = getMockData();
+  let mockSchemaById = getMockSchemaById();
+
+  return (
+    <StructuredDataEngineOutput
+      className={styles.outputViewRoot}
+      data={mockEngineOutput}
+      schemaById={mockSchemaById}
+      engines={engines}
+      selectedEngineId={'engineOfInterest'}
+      onEngineChange={onEngineSelected}
```

Comment:
I think the correct way to do something like this is action knobs https://github.com/storybooks/storybook/tree/master/addons/actions